travis/install_pg

```diff
@@ -4,6 +4,13 @@
 
 set -eux
 
+# If USE_CUSTOM_PG_BUILD variable is set, we will compile PostgreSQL from the source.
+if [ "${USE_CUSTOM_PG_BUILD-0}" != "0" ]; then
+  install_custom_pg
```

Comment:
Rather than having one script call the other, can you just call _both_ in the `.travis.yml` file, and then have each have the "short-circuit" logic at the top to do the right thing?

This is how `install_citus` and `install_pg` work and we used to just call both.